install-dev/upgrade/sql/1.7.3.1.sql

```diff
@@ -0,0 +1,4 @@
+SET SESSION sql_mode = '';
+SET NAMES 'utf8';
+
+ALTER TABLE `PREFIX_order_detail` DROP KEY product_id, ADD KEY product_id (product_id, product_attribute_id);
```

Comment:
If you're on the develop branch, this should be in the file 1.7.4.0.sql

Reply:
ah, I'll move it in 1.7.3.x branch, thanks!